application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -211,6 +211,10 @@ private Optional<String> prepareChatGptQuestion(ThreadChannel threadChannel,
             .min(MAX_QUESTION_LENGTH - questionBuilder.length(), originalQuestion.length()));
 
         questionBuilder.append(originalQuestion);
+
+        questionBuilder.append(
+                ". If possible, get, maximum, 5 top links from reliable websites as references in markdown syntax. Put this message on top of the links list \"Here are some links that may help :\".");
```

Comment:
maybe put the header message manually in embed

Reply:
What do you mean ? I don't see anything called "embed" in `prepareChatGptQuestion()`

Reply:
> I don't see anything called "embed" in `prepareChatGptQuestion()`
Right coz this is just calling the chatgpt API, see [HelpSystemHelper#constructChatGptAttempt](https://github.com/Together-Java/TJ-Bot/blob/develop/application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java#L121) for rest of the logic

Reply:
I think that's not possible, because it should be part of the question sent to chatgpt : `chatGPTAnswer = chatGptService.ask(question, context);` and this call is before generating the embed

Reply:
> I think that's not possible

not with the current approach, u'll have to ask it twice. So, ig leave it like this?